src/CC0-1.0.xml

```diff
@@ -6,6 +6,9 @@
          <crossRef>http://creativecommons.org/publicdomain/zero/1.0/legalcode</crossRef>
       </crossRefs>
       <titleText>
+         <optional>
+            <p>Creative Commons Legal Code</p>
+         </optional>
          <p>Creative Commons CC0 1.0 Universal</p>
```

Comment:
I don't think we want to duplicate “Creative Commons” in this line, see [here][1].

[1]: https://github.com/spdx/license-list-XML/pull/489#discussion_r153060223